go.mod

```diff
@@ -3,49 +3,69 @@ module github.com/kubernetes-csi/external-snapshotter/v4
 go 1.15
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.3.0
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/golang/mock v1.4.3
-	github.com/golang/protobuf v1.4.2
-	github.com/google/gofuzz v1.1.0
-	github.com/imdario/mergo v0.3.9 // indirect
+	github.com/go-logr/logr v0.3.0 // indirect
+	github.com/golang/mock v1.4.4
+	github.com/golang/protobuf v1.4.3
+	github.com/google/go-cmp v0.5.4 // indirect
+	github.com/google/gofuzz v1.2.0
+	github.com/googleapis/gnostic v0.5.3 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.9.0
-	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
+	github.com/kubernetes-csi/csi-test/v4 v4.0.2
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.0.0
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.8.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.10.0
-	github.com/spf13/cobra v1.0.0
-	google.golang.org/grpc v1.29.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v0.19.1
-	k8s.io/component-base v0.19.0
-	k8s.io/klog/v2 v2.3.0
-	k8s.io/kubernetes v1.19.0
+	github.com/prometheus/common v0.15.0
+	github.com/spf13/cobra v1.1.1
+	golang.org/x/crypto v0.0.0-20201208171446-5f87f3452ae9 // indirect
+	golang.org/x/net v0.0.0-20201209123823-ac852fbbde11 // indirect
+	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5 // indirect
+	golang.org/x/sys v0.0.0-20201214095126-aec9a390925b // indirect
+	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
+	golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 // indirect
+	google.golang.org/appengine v1.6.7 // indirect
+	google.golang.org/genproto v0.0.0-20201211151036-40ec1c210f7a // indirect
+	google.golang.org/grpc v1.34.0
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/api v0.20.0
+	k8s.io/apimachinery v0.20.0
+	k8s.io/client-go v1.5.1
```

Comment:
This is suspicious, I would expect `v0.20.0`

Reply:
Yeah, this looks weird.  It's picked up automatically and it is a valid release: https://github.com/kubernetes/client-go/releases/tag/v1.5.1

Reply:
Fixed.